- replaces inflector by its dotnet standard verssion

baywet opened the pull request:
inflector hasn't been upgraded to dotnet standard. However the community put together a fork on dotnet standard that's more popular than the original thing.

MIchaelMainer reviewed (dismissed):
A fork of a fork. Did a quick walk through the commit history since the original fork.

zengin reviewed (approved):
> inflector hasn't been upgraded to dotnet standard. However the community put together a fork on dotnet standard that's more popular than the original thing.

Love this about open source!

baywet commented:
@MIchaelMainer @zengin I've cleaned up the static references and replaced by static instances with culture information. This is good for final review.

zengin commented on src/GraphODataTemplateWriter/CodeHelpers/PHP/TypeHelperPHP.cs:
```diff
@@ -188,7 +188,7 @@ public static string GetPHPNamespace(this string @namespace, string prefix = "")
             pieces.AddRange(@namespace.Split('.').ToList());
 
-            var pascalCasePieces = pieces.Select(piece => piece.Pascalize());
+            var pascalCasePieces = pieces.Select(piece => piece.ToPascalize());
```
old name makes more sense :)

baywet replied:
I agree, however old name maps directly to the inflector extension method when the "new" one maps to our own extension methods. This is the only place as far as I remember that was diverging. I think the original idea behind using our own extension method was to encapsulate inflector.

zengin replied:
Yeah, we should probably rename our extension method (not in this PR) to something like: `ToPascalCase()`.
